WooCommerce/Classes/ViewModels/ProductDetailsCellViewModel.swift

```diff
@@ -1,15 +1,37 @@
 import Foundation
 import Yosemite
 
+protocol AnyAttributeDisplayable {
+    var name: String { get set }
+    var value: String { get set }
+    var nameOrValue: String { get }
+}
```

Comment:
Do you have in mind some functionality for this protocol?  I don't see it being used anywhere! :-)

Reply:
Good point, I'll remove the protocol!